sdk/core/azure-core-tracing-opentelemetry/README.md

```diff
@@ -40,11 +40,13 @@ from azure.core.tracing.ext.opentelemetry_span import OpenTelemetrySpan
 
 settings.tracing_implementation = OpenTelemetrySpan
 
+# In the below example, we use a simple console exporter, uncomment these lines to use
+# the Azure Monitor Exporter.
 # Example of Azure Monitor exporter, but you can use anything OpenTelemetry supports
-from azure_monitor import AzureMonitorSpanExporter
-exporter = AzureMonitorSpanExporter(
-    instrumentation_key="uuid of the instrumentation key (see your Azure Monitor account)"
-)
+# from azure_monitor import AzureMonitorSpanExporter
+# exporter = AzureMonitorSpanExporter(
+#     instrumentation_key="uuid of the instrumentation key (see your Azure Monitor account)"
+# )
```

Comment:
Why we comment this out?

Reply:
just want to make sure users don't need to install azure monitor additionally to run the example.